internal/oidc/staticclient/pinnipedcli.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package staticclient
```

Comment:
Does the static approach put us in a worse place for when we want to define another client?  Should we just create a struct with all the fields we need and then have a singleton for the `pinniped-cli`?